python/plugins/db_manager/db_plugins/connector.py

```diff
@@ -188,12 +188,11 @@ def _get_cursor_columns(self, c):
     @classmethod
     def quoteId(self, identifier):
         if hasattr(identifier, '__iter__') and not isinstance(identifier, str):
-            ids = list()
-            for i in identifier:
-                if i is None or i == "":
-                    continue
-                ids.append(self.quoteId(i))
-            return u'.'.join(ids)
+            return u'.'.join(
```

Comment:
```suggestion
            return '.'.join(
```

While reworking this, the `u` prefix is no longer needed.